0007-configuration-management-extension/README.md

```diff
@@ -33,51 +37,75 @@ It is intended to manage endpoint configuration distribution.
     Possible solutions:
     - Send only updates. (e.g., using JSON Patch format, defined in [RFC 6902](https://tools.ietf.org/html/rfc6902).)
 
-## Use cases
+4. _Endpoint can actively reject a configuration._
+
+   Examples are:
+   - the endpoint is unable to process the configuration.
+   - the configuration is ill-formated, or pre-conditions are not met.
+
+   Possible solutions:
+   - Send the status result of the configuration application (either success or failure) back to the server.
+
+# Use cases
 
-### UC1: Configuration request
+## UC1: Configuration request
 Configuration delivery by request. The endpoint should be able to receive latest configuration from CMX by request.
 
-### UC2: Configuration push
+## UC2: Configuration push
 Configuration delivery is initiated by the server. The endpoint should receive latest configuration when it connects to the server if this configuration wasn't applied yet.
 
-## Design
+# Design
 The extension follows client-initiated request/response pattern described in 1/KP, which nicely maps to both MQTT and CoAP.
 
 There are two resources provided by the extension.
 
 - `/<endpoint_token>/config/json` is used for subscribing and receiving a config.
```

Comment:
How should endpoint distinct between a configuration that was sent as a response to config request and those sent as a push from server side?